client/components/theme/test/index.jsx

```diff
@@ -6,20 +6,22 @@
 /**
  * External dependencies
  */
-import { assert } from 'chai';
-import { identity } from 'lodash';
 import React from 'react';
-import TestUtils from 'react-dom/test-utils';
 import ReactDom from 'react-dom';
 import sinon from 'sinon';
+import TestUtils from 'react-dom/test-utils';
+import { assert } from 'chai';
+import { identity } from 'lodash';
+import { parse } from 'url';
+import { shallow } from 'enzyme';
 
 /**
  * Internal dependencies
  */
 import { Theme } from '../';
 
-jest.mock( 'components/popover/menu', () => require( 'components/empty-component' ) );
-jest.mock( 'components/popover/menu-item', () => require( 'components/empty-component' ) );
+jest.mock( 'components/popover/menu', () => 'components--popover--menu' );
+jest.mock( 'components/popover/menu-item', () => 'components--popover--menu-item' );
```

Comment:
What kind of sorcery is this? 😮

Reply:
I'm not sure whether this should remain or be reverted actually.

I did this thinking about the snapshot, but these components are not included in the shallow render so they're irrelevant.

This is a nice trick for mocks included in snapshots and doesn't harm anything here (apparently). See #19037 and #19100 for extended discussion.

TL;DR is that React will happily inject these into the DOM as elements: `<components--popover--menu any="props">Any children…</components--popover--menu>`, which is quite descriptive when compared to the render of `empty-component` (`<div />`).